src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.spec.ts

```diff
@@ -1,36 +1,55 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ComponentFixture, fakeAsync, flushMicrotasks, TestBed } from '@angular/core/testing';
 import { PagesWalletsMyWalletsComponent } from './pages-wallets-my-wallets.component';
 import { MaterialModule } from '../../../material.module';
 import { PagesWalletsMyWalletsService } from './pages-wallets-my-wallets.service';
 import SpyObj = jasmine.SpyObj;
+import createSpy = jasmine.createSpy;
 import createSpyObj = jasmine.createSpyObj;
-import { Subject } from 'rxjs';
+import { of, Subject } from 'rxjs';
 import { MyWallet } from './pages-wallets-my-wallet.model';
 import { IWalletApiResponse } from '../../../domains/wallets/domains.wallets.types';
-import { WALLET_RESP_MOCK } from 'src/app/domains/wallets/domains.wallets.mocks';
+import { WALLET_INSTANCE_MOCK, WALLET_RESP_MOCK } from 'src/app/domains/wallets/domains.wallets.mocks';
+import { SystemNotificationsService } from 'src/app/common/utils/system-notifications/system-notifications.service';
+import { By } from '@angular/platform-browser';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
+import { WalletFormModalComponent } from './wallet-form-modal/wallet-form-modal.component';
 
 describe('PagesWalletsMyWalletsComponent', () => {
```

Comment:
If you will run this test you will see errors in the terminal that need to be fixed:
```
ERROR: 'NG0303: Can't bind to 'type' since it isn't a known property of 'app-system-message' (used in the 'PagesWalletsMyWalletsComponent' component template).
1. If 'app-system-message' is an Angular component and it has the 'type' input, then verify that it is a part of an @NgModule where this component is declared.
2. If 'app-system-message' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
```